interface/patient_file/morphine_list.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Morphine List
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+require_once("morphine_helper.php");
+use OpenEMR\Core\Header;
+
+$showList = getList();
+
+if(!empty($_POST['drugname'])){
+	
+	$id = filter_input(INPUT_POST, 'id');
+	$drugname = trim(filter_input(INPUT_POST, 'drugname'));
+	$multiplier = trim(filter_input(INPUT_POST, 'multiplier'));
+	$days = filter_input(INPUT_POST, 'days');
+    saveEntry($id, $drugname, $multiplier, $days);
+	echo "<meta http-equiv='refresh' content='0'>";
+}
+
+if(!empty($_GET['update']))
+{
+	$id = filter_input(INPUT_GET, 'update');
+	$updateData = updateDrug($id);
+	
+}
+
+if(!empty($_GET['delete']))
+{
+	$id = filter_input(INPUT_GET, 'delete');
+	deleteEntry($id);
+	echo "<meta http-equiv='refresh' content='0'>";
+}
+
+
+?>
+
+<html>
+<head>
+
+<title><?php echo xlt('Morphine List Manager'); ?></title>
+<?php Header::setupHeader(); ?>
+
+</head>
+<body class="body_top">
+<div class="container">
+<h1><?php echo xlt('Morphine List'); ?></h1>
+<p>
+<?php echo xlt('Please enter the main word of the drug only. For instance') ?>, <b> 
+<?php echo xlt('Embeda') ?></b> <?php echo xlt('can be prescribed as') ?> <b>
+<?php echo xlt('Embeda ER') ?></b>. 
+<?php echo xlt('Only enter') ?> <b><?php echo xlt('Embeda') ?></b>.<br> 
+<?php echo xlt('The system will look for all variation before and after the word Embeda') ?></p>
+<p><?php echo xlt('The Days field is defaulted to 30') ?>. </p>
+<h3><?php echo xlt('Add to list:'); ?></h3>
+<form method="POST" action="morphine_list.php" class="form-inline"  >
```

Comment:
need to deal with session stuff by placing below in above form tag:
``````
onsubmit="return top.restoreSession()"
``````